plugins/modules/openvswitch_db.py

```diff
@@ -109,6 +110,41 @@
     col: tag
     value: 10
     database_socket: unix:/opt/second.sock
+
+# Get interface statistics
+- openvswitch.openvswitch.openvswitch_db:
+    state: read
+    table: interface
+    record: ifname
+    col: statistics
+
+# Get tx_packets value
+- openvswitch.openvswitch.openvswitch_db:
+    state: read
+    table: interface
+    record: ifname
+    col: statistics
+    key: tx_packets
+
+# Get mtu value
+- openvswitch.openvswitch.openvswitch_db:
+    state: read
+    table: interface
+    record: ifname
+    col: mtu
+"""
+
+RETURN = """
+commands:
+    description: List of commands sent
+    returned: when state is get
```

Comment:
Shouldn't this be `read`?

Reply:
Yes. Initially, I used the word "get",  later changed it to "read". Missed updating that here. I will update it. Thanks